packages/api-client/src/auth/PreKeyBundle.ts

```diff
@@ -18,10 +18,19 @@
  */
 
 import {ClientPreKey} from '../auth/';
+import {QualifiedUser} from '../user';
 
-export interface PreKeyBundle {
+type PreKeyBundleBase = {
   /** PreKeys of all clients */
   clients: ClientPreKey[];
+};
+
+export type PreKeyBundle = PreKeyBundleBase & {
   /** The user ID */
   user: string;
-}
+};
+
+export type QualifiedPreKeyBundle = PreKeyBundleBase & {
```

Comment:
What's the benefit of this style over
```ts
export interface QualifiedPreKeyBundle extends PreKeyBundleBase {
```
?

Reply:
No benefit. I will stick with interfaces.